staging/src/k8s.io/kubectl/pkg/cmd/apply/patcher.go

```diff
@@ -164,6 +170,27 @@ func (p *Patcher) patchSimple(obj runtime.Object, modified []byte, namespace, na
 	}
 
 	patchedObj, err := p.Helper.Patch(namespace, name, patchType, patch, nil)
+
+	if err != nil {
+		return nil, nil, errors.Wrap(err, "patch request failed")
+	}
+
+	patchedMeta, err := meta.Accessor(patchedObj)
+
+	if err != nil {
+		// Ignore error? request did go through...
+		return patch, patchedObj, nil
```

Comment:
RFC: I was sure what best behavior would be if the returned object has no ObjectMeta. In this case, it is treated as non-fatal. please advise

Reply:
AFAICS, this does not break functionality and seems valid. Because https://github.com/kubernetes/kubernetes/blob/e02a26d74a2e346b4a5607c99180e5b748923e5f/staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go#L632 handles the cases where patchedObject does not include meta fields.

But for me the question is; in which circumstances, patchedObject does not contain any meta field?, can it be possible?

Reply:
But from the point of object changed/unchanged discrimination, I think we can assume that object is **changed** because we don't know?